web/src/cypress/integration/admin.ts

```diff
@@ -63,23 +75,31 @@ function testAdmin(screen: ScreenFormat) {
       cy.get('input[name="General.PublicURL"]')
         .clear()
         .type(newVal1)
-      cy.get('input[name="Mailgun.EmailDomain"]').type(newVal2)
+        .should('have.value', newVal1)
+      cy.get('input[name="Mailgun.EmailDomain"]')
+        .type(newVal2)
+        .should('have.value', newVal2)
       cy.get('button[data-cy="save"]').click()
 
+      // save dialog
       cy.get('li')
```

Comment:
Should we use `.should('contain'` before the `.contains` for these?

Reply:
> `.contains()` will automatically retry until the element(s) exist in the DOM.

I think as long as we verify that the list has loaded and is present before we move on, we should be good